commands/update-check.go

```diff
@@ -30,18 +30,11 @@ func GetVersionPrinter() func(*cli.Context) {
 //returns a string indicating the new version if available
 func updateCheck(configFile string) string {
 	res := resources.InitResources(configFile)
-	deltaPtr := res.Config.S.UserConfig.UpdateCheckFrequency
+	delta := res.Config.S.UserConfig.UpdateCheckFrequency
```

Comment:
This removed code was to allow UpdateCheckFrequency to have a default value of 14.  This is no longer needed and would have been unsustainable to do for every future addition.